common/djangoapps/student/management/tests/test_bulk_change_enrollment.py

```diff
@@ -15,8 +16,11 @@ class BulkChangeEnrollmentTests(SharedModuleStoreTestCase):
 
     def setUp(self):
         super(BulkChangeEnrollmentTests, self).setUp()
+        patcher = patch('student.models.tracker')
```

Comment:
Are we using this mock object in multiple tests? If we don't then we should just add `@patch('student.models.tracker')` decorator to only the test who wants to mock it.

```
     @patch('student.models.tracker')
     @ddt.data(('audit', 'honor'), ('honor', 'audit'))
     @ddt.unpack
     def test_bulk_convert(self, from_mode, to_mode, mock_tracker):
         ... 
         mock_tracker.emit.assert_has_calls(
                     ...
         ...
```